lib/io/rdf/sparql/sparql-query-rdf-source.ts

```diff
@@ -0,0 +1,12 @@
+import {HttpFetch} from "../../fetch/fetch-api";
+import {RdfMemorySource} from "../rdf-source-base";
+import {fetchRdfQuads} from "../http/http-adapter";
+
+export class SparqlQueryRdfSource extends RdfMemorySource {
+
+  async query(httpFetch: HttpFetch, endpoint: string, query: string): Promise<void> {
```

Comment:
Passing httpFetch and endpoint here means, that you plan to use this source with different httpFetch implementations and endpoints, is that true? This is inconsistent with other sources, such as RdfSparqlSource  that is fixed to given httpFetch and endpoint. 
Do you have a use case to keep a single source instance around and use it with different httpFetch and endpoints?

Reply:
No, sorry. I've fixed that. Your approach introduced in RdfSparqlSource is better.